plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -235,6 +236,16 @@ public String toString()
     {
         StringBuilder builder = new StringBuilder();
         builder.append(relationHandle);
+        if (constraint.isNone()) {
+            builder.append(" constraint=FALSE");
```

Comment:
Is it testable?

Reply:
hopefully it's not, as such TS should be pruned away